ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -348,6 +348,17 @@ static void setForceSyncNum(RaftProperties properties, int forceSyncNum) {
       setInt(properties::setInt, FORCE_SYNC_NUM_KEY, forceSyncNum);
     }
 
+
+    String ASYNC_FLUSH_ENABLE_KEY = PREFIX + ".async.flush";
```

Comment:
We can define it to this:
`String ASYNC_FLUSH_ENABLE_KEY = PREFIX + ".async.flush.enable";`

Reply:
ok, i will change it